README.md

```diff
@@ -0,0 +1,52 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```

Comment:
- [ ] Kindly put the name of your project here so as to make your README file more professional 👍

Reply:
Thank you . Updated. 

![title](https://user-images.githubusercontent.com/40554384/160159055-f0d1ae3e-3477-40d3-ba6d-d4070603b16f.PNG)